config/sources/families/rk3399.conf

```diff
@@ -20,6 +20,17 @@ if [[ $BOARD == roc-rk3399-pc ]]; then
 
 	BOOT_USE_MAINLINE_ATF=yes
 
+elif [[ $BOARD == helios64 ]]; then
+
+	if [[ $BRANCH == legacy ]]; then
```

Comment:
I am glad it was so easy to customise u-boot scenario between branches ;-)
Out of curiosity... did you encounter any issues with mainline ATF in `legacy`?

Reply:
yes. now it is easy to switch the u-boot.
I don't remember if there is any issue with mainline ATF but so far suspend and wakeup only works with legacy kernel (4.4)  and rockchip binaries (ddr, miniloader & ATF).

Reply:
> so far suspend and wakeup only works with legacy kernel (4.4) and rockchip binaries (ddr, miniloader & ATF)

Oh, that is an interesting insight. Thanks for sharing!